Classes/Aspects/AugmentationAspect.php

```diff
@@ -114,7 +114,15 @@ public function contentElementAugmentation(JoinPointInterface $joinPoint)
         $attributes['data-__neos-node-contextpath'] = $node->getContextPath();
         $attributes['data-__neos-fusion-path'] = $fusionPath;
 
-        return $this->htmlAugmenter->addAttributes($content, $attributes);
+        // Make all added attributes exclusive to prevent the data of two different nodes to be concatenated
```

Comment:
ahh i see we have this feature already lol :D Could we please use a variable `$exclusiveAttributes` so the comment is pinpointed to exactly that?

And also does it make sense to add a small unit test here? On the otherhand i believe the `htmlAugmenter` is well tested so no problem :)

Reply:
I adjusted the comment a bit.